.travis.yml

```diff
@@ -1,15 +1,16 @@
 sudo: required
-dist: trusty
+dist: jammy
 language: node_js
 node_js:
-  - '12'
+  - '14.20.1'
```

Comment:
Why using NodeJS 14.x on continuous integration and then using version 16.x on file for node version manager? I would go also for the 16.x version on TravisCI considering this is the latest LTS.

Reply:
Because of a problem with Systelab-Preferences I prefered to use the same version for all Systelab libraries